validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+# RANCHERINT Schemas
+
+## Test Suite: Longhorn
+
+### Fresh Longhorn Installation via Rancher App Catalog
+
+Verify Longhorn can be successfully installed through Rancher's app catalog with default settings on a clean Kubernetes cluster.
+
+| Step Number | Action                                            | Data | Expected Result                                             |
+|------|---------------------------------------------------|------|-------------------------------------------------------------|
+| 1 | Log into Rancher UI with cluster admin privileges |      | Rancher dashboard loads successfully                        |
+| 2 | Navigate to target cluster in Rancher dashboard   |      | Cluster view displays                                       |
+| 3 | Click "Apps" in the left navigation menu          |      | Apps section opens                                          |
+| 4 | Click "Charts" to access the application catalog  |      | Charts catalog displays                                     |
+| 5 | Search for "Longhorn" in the search bar           |      | Longhorn chart appears in results                           |
+| 6 | Select the Longhorn chart from the results        |      | Longhorn installation page opens                            |
+| 7 | Review default settings and click "Install"       |      | Installation begins                                         |
+| 8 | Monitor installation progress in the Apps section |      | Installation completes within 10 minutes                    |
+| 9 | Verify all Longhorn pods are in Running state     |      | All pods show "Running" status in longhorn-system namespace |
+| 10 | Check Longhorn UI accessibility through Rancher |      | Longhorn UI opens when clicking Longhorn button  |
+| 11 | Verify default storage class creation     |      | `longhorn` storage class exists and is available |
```

Comment:
this is a discussion point for the team; should we write steps/actions as steps in the UI, or steps that we take in automation? i.e. step 11 vs. steps 2-7. step 11 is a verify step, which implies that the user in the UI would navigate to storage classes, click on longhorn, and verify/check its in an active state/that it exists. Similarly in automation, this would imply you do the same thing but via a steve call.

For the tests/schemas I wrote, I did so with this approach. But steps 2-7 explicitly call out UI steps. I would lean toward step 11 vs. explicit UI steps, since we will ultimately be automating all/most of these. 

wdyt?